hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/FileUpdateSuite.java

```diff
@@ -225,21 +225,18 @@ public HapiSpec notTooManyFeeScheduleCanBeCreated() {
         final var denom = "fungible";
         final var token = "token";
         return defaultHapiSpec("OnlyValidCustomFeeScheduleCanBeCreated")
-                .given(fileUpdate(APP_PROPERTIES)
-                        .payingWith(GENESIS)
-                        .overridingProps(Map.of(MAX_CUSTOM_FEES_PROP, "1")))
+                .given(overriding(MAX_CUSTOM_FEES_PROP, "1"))
```

Comment:
Any reason why we should change `payingWith(GENESIS)`?

Reply:
`overriding()` always uses `GENESIS`